### ISSUE TITLE
Error reported when deleting webhooks with multiple PipelineRuns

### ISSUE THREAD

**Issue by jessm12** [tektoncd/experimental#201]
# Expected Behavior
When using the experimental webhooks-extension for the Dashboard, deleting a webhook and its associated `PipelineRuns` completes successfully and reports as a success

# Actual Behavior
<img width="1158" alt="Screenshot 2019-08-14 at 11 01 51" src="https://user-images.githubusercontent.com/32932155/63015319-83f1c380-be88-11e9-8fd4-6fca9ff5011c.png">
@dibbles first observed this problem when deleting two webhooks at once which both had a large number of runs I was able to see the same in my environment with a similar scenario.

The deletion of both the webhooks themselves and the `PipelineRuns` appeared to be successful though did seem to take a while to complete. 

# Steps to Reproduce the Problem

1. Create multiple webhooks and trigger them multiple times
2. Try to delete the webhooks and their associated `PipelineRuns`
3. Observe the error as shown above 

# Additional Info
We aren't sure of the cause of this issue but suspect it could be some form of service timeout 🤔

**Comment by a-roberts** [comment]
Picked this up, got two Git repos with webhooks triggering the same pipeline (using the 0.1.1 release) on OpenShift 3.11.

Clicked to delete both webhooks, confirmed I see the error being reported and yet all is well

```
An error occurred deleting webhook(s).

Check the webhook(s) existed and both the dashboard and extension pods are healthy.
```

No errors in the backend, when I implemented this I added a bunch of delete promises with a 500ms timeout (potentially for *everything* to be done and not just for each promise, but I'll check). Definitely to do with the frontend handling of it that I added, so on the case, and it's the same deletion code and table on master so I'm sure it'll be seen there.
